cryptography/src/bls12381/dkg.rs

```diff
@@ -1361,31 +1360,26 @@ pub fn deal_anonymous<V: Variant>(
     (output.public().clone(), shares.values().to_vec())
 }
 
-#[cfg(test)]
-mod test {
+#[cfg(any(feature = "fuzz", test))]
```

Comment:
When you discussed this pattern this morning, I was afraid it required feature flags inline. I think (at a high-level) what you have here is totally fine.